src/lib.rs

```diff
@@ -287,16 +289,14 @@ impl fmt::Display for Error {
 impl StdError for Error {
     fn description(&self) -> &str {
         match *self {
-            Error::Api(ref e) => {
-                e.error_description.as_ref().unwrap_or(&e.error)
-            },
+            Error::Api(ref e) => e.error_description.as_ref().unwrap_or(&e.error),
```

Comment:
Same as the function signature, anything beyond 80 characters should be truncated onto the next lines.